packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -86,6 +103,23 @@ const BuySellForm = props => {
         }
     };
 
+    const onSubmitAdTransaction = async (...args) => {
+        if (rate_type === ad_type.FLOAT) {
+            const updated_market_rate = await fetchCurrentMarketRate();
+            if (market_rate_on_mount !== updated_market_rate) {
+                buy_sell_store.setShouldShowPopup(false);
+                // TODO: Will remove this once https://github.com/binary-com/deriv-app/pull/5141 PR is merged
+                setTimeout(() => {
+                    buy_sell_store.setShowRateChangedPopup(true);
+                }, 250);
+            } else {
```

Comment:
If both `else` statements, this one and the one below do the same things, we can just combine it into a single condition

Reply:
Since there is an aPI call that is to be executed only when Ad type is Float, I have split them into sub condition